src/services/qAccService.test.ts

```
@@ -88,29 +92,48 @@ describe('qAccService', () => {
   });
   afterEach(async () => {
     // Clean up the database after each test
     await ProjectRoundRecord.delete({});
     await Donation.delete({ projectId: project.id });
     await EarlyAccessRound.delete({});
-    await QfRound.delete(qfRound1.id);
+    await QfRound.delete({});
   });
 
-  it('should return correct value for single early access round', async () => {
+  it.only('should return correct value for single early access round', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)